libqtile/widget/mpris2widget.py

```diff
@@ -275,18 +269,31 @@ def _task_callback(self, task: asyncio.Task) -> None:
         if message.message_type != MessageType.METHOD_RETURN:
             logger.warning("Unable to send command to player.")
 
-    def cmd_play_pause(self) -> None:
+    @expose_command()
+    def play_pause(self) -> None:
         """Toggle the playback status."""
         self._player_cmd("PlayPause")
 
-    def cmd_next(self) -> None:
+    @expose_command()
+    def next(self) -> None:
         """Play the next track."""
         self._player_cmd("Next")
 
-    def cmd_previous(self) -> None:
+    @expose_command()
+    def previous(self) -> None:
         """Play the previous track."""
         self._player_cmd("Previous")
 
-    def cmd_stop(self) -> None:
+    @expose_command()
+    def stop(self) -> None:
         """Stop playback."""
         self._player_cmd("Stop")
+
+    @expose_command()
+    def info(self):
+        """What's the current state of the widget?"""
+        d = base._TextBox.info(self)
+        d.update(
+            dict(displaytext=self.displaytext, isplaying=self.is_playing, player=self.player)
```

Comment:
How come `displaytext` was added to this info? `base._TextBox.info(self)` still provides `text` but the test for this widget has been updated to check `displaytext` instead of `text`

Reply:
This was old. Good spot!